modules/etl/talend/talend-definition/src/main/java/com/liferay/talend/connection/LiferayConnectionProperties.java

```diff
@@ -281,6 +298,22 @@ public ValidationResult validateTestConnection() {
 		"testConnection", "Test Connection");
 	public Property<String> userId = PropertyFactory.newString("userId");
 
+	public enum LoginType {
+
+		Basic("Basic authentication");
```

Comment:
`Basic authentication` or `Basic Authentication`?

Reply:
I'll rename to `Basic Authentication`